view/frontend/web/js/view/form/fields.js

```diff
@@ -72,8 +72,8 @@ define([
                     streetElement.enable();
                     cityElement.enable();
                 } else {
-                    streetElement.disable();
```

Comment:
Let er even goed op wat je mee commit.
Dit hadden we bij Buckaroo gedaan om ervoor te zorgen dat die adresvelden disabled werden weet je nog? Dit wil je niet in je commit mee hebben.